--- ./package/tools	2014-12-25 23:53:44.111603808 -0800
+++ ./package/tools	2014-12-25 23:53:49.134937420 -0800
@@ -14,17 +14,17 @@
 	echo ar > ar
 	echo rc > arflags
 fi
-if type >/dev/null clang++
+if type >/dev/null g++
 then
-	echo clang++ > cxx
+	echo g++ > cxx
 	echo > cppflags
-	echo -g -pthread -std=gnu++11 -Os -Wall -Wextra -Wshadow -Wcast-qual -Wsynth -Woverloaded-virtual -Wcast-align -integrated-as > cxxflags
+	echo -g -pthread -std=gnu++11 -Os -Wall -Wextra -Wshadow -Wcast-qual -Wsynth -Woverloaded-virtual -Wcast-align > cxxflags
 	echo -g -pthread > ldflags
-elif type >/dev/null g++
+elif type >/dev/null clang++
 then
-	echo g++ > cxx
+	echo clang++ > cxx
 	echo > cppflags
-	echo -g -pthread -std=gnu++11 -Os -Wall -Wextra -Wshadow -Wcast-qual -Wsynth -Woverloaded-virtual -Wcast-align > cxxflags
+	echo -g -pthread -std=gnu++11 -Os -Wall -Wextra -Wshadow -Wcast-qual -Wsynth -Woverloaded-virtual -Wcast-align -integrated-as > cxxflags
 	echo -g -pthread > ldflags
 elif type >/dev/null owcc
 then
