diff --git a/window-copy.c b/window-copy.c
index 507461d..6382aa8 100644
--- a/window-copy.c
+++ b/window-copy.c
@@ -763,7 +763,8 @@ window_copy_command(struct window_pane *wp, struct client *c, struct session *s,
 		if (strcmp(command, "copy-pipe") == 0) {
 			if (s != NULL) {
 				window_copy_copy_pipe(wp, s, NULL, argument);
-				window_pane_reset_mode(wp);
+				window_copy_clear_selection(wp);
+				window_copy_redraw_screen(wp);
 			}
 		}
 		if (strcmp(command, "goto-line") == 0)
